iris_doc/language_specification.py

```diff
@@ -143,9 +144,9 @@ def __parseJson(self, jsonContent: str):
                 parentElement = next(
                     (e for e in elementsCopy if e['id'] == f"{tmpType}_{newName1}"), None)
                 if parentElement:
-                    newName1 = parentElement['name'].lower()
+                    newName1 = parentElement['name'].lower().translate(str.maketrans("", "", "()"))
```

Comment:
Is there any example of why the name needs `maketrans`?

Reply:
Same as the extensionName field, for properties and methods inside AgoraRtcEngine(Ex)

Reply:
Removed